sdk/tables/azure-data-tables/azure/data/tables/aio/_table_service_client_async.py

```diff
@@ -0,0 +1,293 @@
+        :return: TableServiceProperties, or the result of cls(response)
+        :rtype: ~azure.data.tables.models.TableServiceProperties
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        timeout = kwargs.pop('timeout', None)
+        try:
+            service_props = await self._client.service.get_properties(timeout=timeout, **kwargs)  # type: ignore
+            return service_properties_deserialize(service_props)
+        except HttpResponseError as error:
+            process_table_error(error)
+
+    @distributed_trace_async
+    async def set_service_properties(
+            self,
+            analytics_logging=None,  # type: Optional[TableAnalyticsLogging]
+            hour_metrics=None,  # type: Optional[Metrics]
+            minute_metrics=None,  # type: Optional[Metrics]
+            cors=None,  # type: Optional[CorsRule]
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        """Sets properties for an account's Table service endpoint,
+        including properties for Analytics and CORS (Cross-Origin Resource Sharing) rules.
+
+       :param analytics_logging: Properties for analytics
+       :type analytics_logging: ~azure.data.tables.TableAnalyticsLogging
+       :param hour_metrics: Hour level metrics
+       :type hour_metrics: ~azure.data.tables.Metrics
+       :param minute_metrics: Minute level metrics
+       :type minute_metrics: ~azure.data.tables.Metrics
+       :param cors: Cross-origin resource sharing rules
+       :type cors: ~azure.data.tables.CorsRule
+       :return: None
+       :rtype: None
+       :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        props = TableServiceProperties(
+            logging=analytics_logging,
+            hour_metrics=hour_metrics,
+            minute_metrics=minute_metrics,
+            cors=cors
+        )
+        try:
+            return await self._client.service.set_properties(props, **kwargs)  # type: ignore
+        except HttpResponseError as error:
+            process_table_error(error)
+
+    @distributed_trace_async
+    async def create_table(
+            self,
+            table_name,  # type: str
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> TableClient
+        """Creates a new table under the given account.
+
+        :param headers:
+        :param table_name: The Table name.
+        :type table_name: ~azure.data.tables._models.Table
+        :return: TableClient, or the result of cls(response)
+        :rtype: ~azure.data.tables.TableClient or None
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        _validate_table_name(table_name)
+
+        table_properties = TableProperties(table_name=table_name, **kwargs)
+        await self._client.table.create(table_properties=table_properties, **kwargs)
+        table = self.get_table_client(table=table_name)
+        return table
```

Comment:
This should be updated as per the sync client

Reply:
Created an issue for this, will address in separate PR